projects/extension/src/containers/Options.tsx

```diff
@@ -175,8 +184,10 @@ const Options: React.FunctionComponent = () => {
           {/** Networks section */}
           <TabsContent activeTab={activeTab}>
             <section>
-              {bg?.uiInterface?.smoldotCrashError ? (
-                <ClientError error={bg?.uiInterface?.smoldotCrashError} />
+              {clientError || bg?.uiInterface.smoldotCrashError ? (
+                <ClientError
+                  error={clientError || bg?.uiInterface.smoldotCrashError || ""}
+                />
```

Comment:
```suggestion
              {clientError !== undefined ? (
                <ClientError
                  error={clientError}
                />
```

Reply:
This will lead to the error appearing ONLY if the popup or options page is open at the moment that the error takes place. If the error happens and after the user opens the pages then no error message will appear.